docs/gettingStarted/setupGenericScheme.md

```diff
@@ -0,0 +1,217 @@
+        "voteOnBehalf": "0x0000000000000000000000000000000000000000",
+        "votersReputationLossRatio": 4,
+        "activationTime": 0
+      }
+
+  // Get address from https://github.com/daostack/migration/blob/master/migration.json
+  const votingMachineAddress = "0xaddress-of-VotingMachine-of-DAO-on-given-network"
+
+  // For eg if you want this Generic Scheme to enable DAO to interact with Bounties Network
+  // then targetContract would be the address of Bounties Network's respective contract
+  const targetContractAddress = "0xaddress-of-contract-this-will-interact-with"
+
+  const avatar = "0xaddres-of-DAO"
+
+  // paramHash will be useful in later step so lets log it
+  const paramHash = genericScheme.methods.initialize(
+      avatar,
+      voteParams,
+      votingMachineAddress,
+      targetContractAddress
+      ).call()
+
+  console.log(paramHash)
+
+  genericScheme.methods.initialize(
+        avatar,
+        voteParams,
+        votingMachineAddress,
+        targetContractAddress
+      ).send()
+```
+
+#### Submit new proposal to Scheme Registrar via Alchemy UI
+
+  1. On Alchmey landing page choose the DAO to which you wish to register the scheme
+  2. Visit DAO's `Home` page and choose `Scheme Registrar`
+  3. Click `New Proposal`, this will open a popup
+  4. Select `Add Scheme` on the popup sidebar (on left)
+  5. Give appropriate title, description, url which describes the proposal
+  6. Put the address of Generic Scheme contract
+
+      **NOTE**: Based on above step this would be either address of UGenericScheme (if registering generic scheme for first time or editing params for existing one) or address of GenericScheme instance deployed in previous step (if registering multiple generic scheme to the DAO)
+
+  7. Enter the `paramHash` you got [here](#set-generic-scheme-to-interact-with-your-contract)
+  8. In permissions section choose `Call genericCall on behalf of`
+  9. Submit Proposal and sign off the transaction
+  10. If the DAO agrees then your Generic Scheme with ability to interact with the `targetContract` will be registered to the DAO
+
+## How to get your Generic Scheme shown in Alchemy
+
+You will have to submit PR to Alchemy repo with details of your scheme. So if this was Generic Scheme to interact with Bounties Network we will create, say, `Bounties.json` file and put [here](https://github.com/daostack/alchemy/tree/dev/src/genericSchemeRegistry/schemes)
+
+```
+  // Bounties.json
+  {
+    "name": "Bounties",
+    "address": {
+      "main": [ "0xtarget-contract-address-on-mainnet" ],
+      "rinkeby": [],
+      "private": []
+    },
+    "actions": [
+      {
+        "id": "this-is-a-method-on-contract",
+        "label": "this will appear on Alchemy UI for submitting proposal for this action",
+        "description": "Describe what this action will do on-chain when proposal is passed by DAO",
+        "notes": "",
+        "fields": [
+          {
+            "label": "label for this field",
```

Comment:
Here the example can continue:
label: "bounty title"
name: name-of-field-in-abi
placeholder: "Osam Bin-Laden - dead or alive"

Reply:
These are some gooood examples 😂